### ISSUE TITLE
RatRig MGN9 - Belt position deviated on Y and Z axis

### ISSUE THREAD

**Issue by pedromelim** [VzBoT3D/Vz-Printhead-Printed#4]
When trying to adapt the Printhead to use on the original MGN12 Carriage from the RatTig, i discovered that the belt positioning on the Vz-Printehead Printed for the RatRig, to use the MGN9H, is off both in Y and Z axis.

I found these using as a guide the supplied V-Core 3 assembly supplied by RatRig.

Vz-PrintHead Printed RatRig MGN9

Measured from the middle of carriage

Front Posts position (Y AXIS) - 17mm
Rear Posts position (Y AXIS) - 15.8mm

Considering V-core 3 assembly and how the belt should wrap around post, position should be:

Front Posts position (Y AXIS)- 16.601mm
Rear Posts position (Y AXIS)- 15.419mm

For the belt position measured from the top of the carriage to the middle of the 9mm belt:

Top Belt position (Z AXIS) - 12.7mm
Bottom Belt position (Z AXIS) - 25.717mm

With MGN12C (Default V-Core 3 Carriage):

Top Belt position (Z AXIS) - 17.6mm
Bottom Belt position (Z AXIS) - 30.6mm

If we consider that the MGN12C does have 3mm more than the MGN9H, then the adjusted position for the Vz-PrintHead Printed RatRig MGN9 should be:

Top Belt position (Z AXIS) - 14.6mm
Bottom Belt position (Z AXIS) - 27.6mm

In summary:

Front Posts - Should be moved 0.399mm closer to the carriage
Rear Posts - Should be moved 0.381mm closer to the carriage

Top Belt Position - Should be lowered by 1.9mm
Bottom Belt Position - Should be lowered by 1.883mm

**Comment by karnadii** [comment]
@pedromelim can you show an image please for the measurement deviation?

**Comment by pedromelim** [comment]
> @pedromelim can you show an image please for the measurement deviation?

All i could do would be to take screenshot measuring in Fusion 360 with some extra calculations. Did not print this as iam trying to remix this for the MGN12C that RatRig includes with the V-core 3.

Would that work?

**Comment by karnadii** [comment]
> > @pedromelim can you show an image please for the measurement deviation?
> 
> All i could do would be to take screenshot measuring in Fusion 360 with some extra calculations. Did not print this as iam trying to remix this for the MGN12C that RatRig includes with the V-core 3.
> 
> Would that work?

that would do.

**Comment by karnadii** [comment]
Fixed, check https://www.printables.com/model/379041-ratrig-vz-printed-printhead-betauntested/files

let me know if there is another issue.
